app/views/purchases/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class='new-container'>
+  <div class='sign-up-details'>
+    <%= link_to category_purchases_path do %>
+      <%= image_tag 'back.png', class: 'back' %>
+    <% end %>
+    <h2 class='centered-text'>Add a transaction</h2>
+  </div>
+
+  <%= form_with(model: [@categories, @new_purchase], url: category_purchases_path(@category.id), scope: :purchase) do |form| %>
+    <div class='form-container'>
+      <div class='field'>
+        <%= form.label :name %>
+        <%= form.text_field :name, autofocus: true, placeholder: 'Enter transaction name here...' %>
+      </div>
+
+      <div class='field'>
+        <%= form.label :amount %>
+        <%= form.number_field :amount, placeholder: 'Enter amount for the transaction...' %>
+      </div>
+      <%= hidden_field_tag 'category_id', @category.id %>
+
```

Comment:
- [ ] A user should be able to select a category on this form when creating a transaction.